drafts/2019-11-06-1900011.adoc

```diff
@@ -54,6 +54,7 @@ Or take a look at our link:/downloads/[Downloads page].
 [link=https://stackoverflow.com/tags/open-liberty]
 image::/img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]
 
+[#pelt]
 == Printing elapsed time message on completion of slow request with Request Timing 1.0
 
 This feature provides a way of monitoring requests with respect to time. In particular, the feature will automatically detect slow and hung requests all while routing the information back to users to assist with diagnosing the cause of the slow or hung requests.
```

Comment:
"...the feature automatically detects..." (remove future tense when talking about what the tech does)